ui/components/app/assets/token-cell/cells/token-cell-secondary-display.tsx

```diff
@@ -41,11 +40,16 @@ export const TokenCellSecondaryDisplay = React.memo(
   }: TokenCellSecondaryDisplayProps) => {
     const t = useI18nContext();
     const isEvm = useSelector(getMultichainIsEvm);
-    const currencyRates = useSelector(getCurrencyRates);
+    const { type, rpcUrl } = useSelector(getProviderConfig);
 
-    const isOriginalTokenSymbol = token.symbol && currencyRates[token.symbol];
+    const isOriginalNativeToken = useIsOriginalNativeTokenSymbol(
+      token.chainId,
+      token.symbol,
+      type,
+      rpcUrl,
+    );
```

Comment:
Using a hook to get the correct original network token symbol for a given native token.